### PR TITLE
feat: create article data tag component

### PR DESCRIPTION
**Pull request by Olaleye-Blessing**
Migrate `DataTag` component and create a storybook's story.

Issue: #5242

### CODE REVIEW

**Review by manishprivet — APPROVED** [review]
LGTM 🚀  Thanks for the first contribution 😄

File: components/Article/DataTag/index.module.scss [hunk 1]
```diff
@@ -0,0 +1,26 @@
+.data-tag {
```
Comment (ovflowd) [inline components/Article/DataTag/index.module.scss]: Can the class names be `camelCse`?

---

cc @manishprivet could you search for StyleLint configs that ensure camelcase? (Also, could you check if Stylelint is working et all? Somehow linting passed on this https://github.com/nodejs/nodejs.org/pull/5236#discussion_r1167789534

**Review by ovflowd — APPROVED** [review]
Really cool! Thank you for your first contribution! :shipit: 🎉 

I left ta minor comment but besides that, this is fine :)

**Review by shanpriyan — APPROVED** [review]
LGTM! Thanks for your first time contribution 🚀

**Review by aymen94 — APPROVED** [review]

**Pull request merged**